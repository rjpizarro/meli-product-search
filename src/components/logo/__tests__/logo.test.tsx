import React from 'react';
import renderer from 'react-test-renderer';
import Logo from '../index'

describe('Logo', () => {
    it('should render correctly ', () => {
        const tree = renderer
            .create(<Logo />)
            .toJSON()

        expect(tree).toMatchSnapshot()
    })
})

