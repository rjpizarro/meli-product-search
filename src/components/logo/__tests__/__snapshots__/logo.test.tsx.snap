// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render correctly  1`] = `
<img
  alt="Mercado Libre"
  sizes=" (max-width: 480px) 53px, 60px, "
  src="logo_ml@2x.png"
  srcSet="logo_ml@2x.png 106w,
                logo_ml.png 53w"
/>
`;
